src/handlers/backport.rs

```diff
@@ -50,13 +50,14 @@ pub(super) async fn parse_input(
         None => return Ok(None),
     };
 
-    // Only handle events when the PR is opened or the first comment is edited
+    // Only handle events when the PR is opened (and it's not a draft) or when the first comment is edited
     let should_check = matches!(event.action, IssuesAction::Opened | IssuesAction::Edited);
-    if !should_check || !event.issue.is_pr() {
+    if !should_check || !event.issue.is_pr() || !event.issue.draft {
```

Comment:
This should be `event.issue.draft`, right? Otherwise you would exclude non-draft PRs :sweat_smile:

Reply:
:sob: